app/templates/crate/versions.hbs

```diff
@@ -16,22 +16,27 @@
   </div>
 </div>
 
-<ul local-class="list">
+<ul local-class="list" class="{{unless this.sortedVersions 'is-empty'}}">
   {{#each this.sortedVersions as |version|}}
     <li>
       <VersionList::Row @version={{version}} local-class="row" data-test-version={{version.num}} />
     </li>
   {{/each}}
 </ul>
-{{#if this.next_page}}
+{{#if this.loadMoreTask.isRunning}}
```

Comment:
should we also hide the `.page-description` element above while `this.loadMoreTask.isRunning && this.sortedVersions.count === 0`?

Reply:
So, this is regarding the initial load (like the 'is-empty' check mentioned above)? I presume this could be achieved via style="{{unless this.sortedVersions 'visibility:hidden'}}" (which could also be implemented using a CSS class and style).

I don't really have a preference on this, but I think the reasoning behind it is to make it more noticeable that it's still loading and is an initial load. So, I'm okay with this.

I'm also wondering if we should prevent the top loading indicator (the yellow one) from completing until the first versions have finished loading.

Reply:
> should we also hide the `.page-description` element above while `this.loadMoreTask.isRunning && this.sortedVersions.count === 0`?

done!